ScnSoft-Education/LeetCodeTraining.Tests/TwoSumSolutionTest.cs

```diff
@@ -0,0 +1,27 @@
+using LeetCodeTraining.TwoSum1;
+
+namespace LeetCodeTraining.Tests
+{
+    public class TwoSumSolutionTest
```

Comment:
Please add //Arrange //Act //Assert comments appropriately.
Also please implement all the same 3 Cases from Test Case tab in this method, from problem's description page
https://leetcode.com/problems/two-sum/.
![image](https://github.com/ArinaSiselina/ScnSoft-Education/assets/101803780/dabac430-ebb2-486f-96d5-b1a962b0471f)